dvc/config_schema.py

```diff
@@ -138,6 +138,7 @@ class RelPath(str):
                     "region": str,
                     "profile": str,
                     "credentialpath": str,
+                    "configpath": str,
```

Comment:
qq - is it different from the `credentialpath`?

Reply:
`credentialpath` is the replacement for `~/.aws/credentials` file, and `configpath` would be the replacement for `~/.aws/config`

Reply:
yep, I see.